_locales/ja/messages.json

```diff
@@ -0,0 +1,14 @@
+{
+  "extension_name": {
+    "message": "Toggle YouTube Comments"
+  },
+  "extension_desc": {
+    "message": "Hides YouTube comments by default"
```

Comment:
Would you like to translate the name and description when we change the webstore's detailed description?

Reply:
I think that the extension name (title) and heading desc should remain in English for now.
name & desc is sufficiently concise in English.
If translated into Japanese, it will be prolix sentences. So it will be difficult to understand for Japanese users 🤔